specs/slack-primitive.md

```
@@ -0,0 +1,301 @@
+  | 'human_no_response' // askQuestion hit timeoutSeconds
+  | 'reply_did_not_match' // got a reply but replyMatch rejected it
+  | 'reply_from_unauthorized_user'
+  | 'rate_limited' // 429, with retry-after honored automatically
+  | 'slack_api_error'; // catch-all, includes upstream message
+```
+
+These match the github-primitive's error-code shape so workflow `onError` handlers can discriminate on `err.code` consistently across primitives.
+
+## 7. Implementation outline
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)